docs/src/gs/install-cluster-api.md

```diff
@@ -52,10 +52,12 @@ allow dynamic-group [your dynamic group name] to manage load-balancers in compar
 
 ## Initialize management cluster
 
-Initialize management cluster and install CAPOCI
+Initialize management cluster and install CAPOCI.
+
+The latest version can be found [on GitHub][capoci-latest-release]
 
    ```bash
-      clusterctl init --infrastructure oci:v0.6.0
+      clusterctl init --infrastructure oci:v0.7.0
```

Comment:
I dont think this will scale, and we may miss this every time. Can we make this just oci and put a note sayin gthat in poroduction use aporticular version and upgrade to aporticular version , or something to that sort?

Reply:
Great idea. I did add the "latest" link so I think that makes sense.